robot/Cumulus/resources/NPSP.py

```diff
@@ -868,7 +868,29 @@ def wait_for_batch_to_complete(self, path, *args, **kwargs):
                     self.selenium.wait_until_element_is_visible(locator)
                     break
                 except Exception:
-                    time.sleep(15)    
+                    time.sleep(15)   
```

Comment:
What would happen if we do not give it a sleep time?

Reply:
same answer as below one and this keyword is going away soon once I update the below keyword in every test case